src/fsharp/fsi/fsi.fs

```diff
@@ -1361,18 +1372,24 @@ type internal FsiDynamicCompiler
                     ccuinfos
             ccuinfos, ty
 
-        let ilTy = convertReflectionTypeToILType reflectionTy
```

Comment:
This part is much clearer if you're using the split view; the next 11 lines were simply placed into a function.